app/models/illinois_rate.rb

```diff
@@ -33,8 +33,7 @@ class IllinoisRate < UuidApplicationRecord
             .where('expires_on is null or expires_on > ?', date.at_beginning_of_month)
             .where('age_bucket >= ? OR age_bucket IS NULL', age)
             .where(region: Illinois::RegionFinder.new(business: business).call)
-            .where(license_type: business.license_type)
-            .order_age_bucket
+            .where(license_type: business.license_type).order_age_bucket
```

Comment:
can you move the order_age_bucket to the next line as it was?

Reply:
done